client/java/src/main/java/com/linecorp/centraldogma/internal/client/AbstractWatcher.java

```diff
@@ -49,8 +55,46 @@
     private static final long MAX_INTERVAL_MILLIS = TimeUnit.MINUTES.toMillis(1);
     private static final double JITTER_RATE = 0.2;
     private static final long WATCH_TIMEOUT_MILLIS = TimeUnit.MINUTES.toMillis(1);
-    private static final Supplier<? extends Backoff> BACKOFF_SUPPLIER =
-            () -> Backoff.exponential(MIN_INTERVAL_MILLIS, MAX_INTERVAL_MILLIS).withJitter(JITTER_RATE);
+
+    private static long nextDelayMillis(int numAttemptsSoFar) {
```

Comment:
Is there any reason that you implemented this logic by yourself?

Reply:
Because it has to be Armeria-free?

Reply:
Ah got it!